decidim-core/lib/tasks/decidim_shakapacker_tasks.rake

```diff
@@ -45,30 +45,36 @@ namespace :decidim do
         @rails/ujs
         turbolinks
       )
-      system! "npm uninstall #{webpacker_packages.join(" ")}"
+      system! "npm uninstall #{shakapacker_packages.join(" ")}"
 
       # Add the Browserslist configuration to the project
       add_decidim_browserslist_configuration
     end
   end
 
   namespace :upgrade do
-    desc "Upgrades Decidim webpacker dependencies in Rails instance application"
-    task webpacker: :environment do
+    desc "Upgrades the shakapacker required npm package"
+    task :shakapacker_npm do
+      system! "npm uninstall shakapacker"
+      system! "npm install --save-dev shakapacker@#{`bundle list | grep shakapacker | awk -F '[()]' '{print $2}'`}"
```

Comment:
Using shell scripting with grep/awk just feel wrong here. Maybe we can do `Shakapacker::VERSION` instead?